factory/runType/sovereignRunTypeComponents.go

```diff
@@ -37,13 +38,15 @@ type ArgsSovereignRunTypeComponents struct {
 	Config                   config.SovereignConfig
 	DataCodec                sovereign.DataCodecHandler
 	TopicsChecker            sovereign.TopicsCheckerHandler
+	ExtraVerifier            process.ExtraHeaderSigVerifierHandler
 }
 
 type sovereignRunTypeComponentsFactory struct {
 	*runTypeComponentsFactory
 	cfg           config.SovereignConfig
 	dataCodec     sovereign.DataCodecHandler
 	topicsChecker sovereign.TopicsCheckerHandler
+	extraVerifier process.ExtraHeaderSigVerifierHandler
```

Comment:
This should be deleted

Reply:
To minimize the conflicts with the next PRs, I created another PR for this at the end of refactor 
https://github.com/multiversx/mx-chain-go/pull/6140